std_msgs/Header header
int16 var1
int16 var2
int16 var3
int16 var4
